### DISCUSSION TITLE
Modules are configured by .dist files instead of .conf

### DISCUSSION THREAD

**Discussion by Zaedon33** [azerothcore/azerothcore-wotlk#18322] [General]
Hi all! first post! I've been using different repacks decided to compile my own server. However, I've noticed that all mods are using the .dist files to configure mods instead of the .conf files. This means every time I compile I need to redo all my configurations in every add. Of course, I have the conf files that I can just copy and paste. However, this seems like an unnecessary hassle. I' certain I have something wrong somewhere. lol

Info:
Operating system: Ubuntu 22.04.3 LTS
AzerothCore rev. unknown 1970-01-01 (Custom Branch for Playerbots)

Auth.conf and worldserver.conf work as they should. In fact, everything works as it should. However, all information I can find is telling me that this is wrong and that the .conf files for modules should be being used instead of the conf.dist files. The server was set up following the guide here on Azerothcre. The only exception is that I used the custom core for playerbots as I prefer playbots to npcbots.

Any help would be much appreciated! Even if it is only to tell me that I'm an idiot. ;) Thanks in advance!

**Comment by virgo77** [comment]
Hi, are your modules .conf files in the same directory as the .conf.dist files, such as: 'modules' folder?
authserver.conf and worldserver.conf must stay in the 'etc' folder, and all the modules .conf files should be in the 'etc/modules' folder.

For me, it's working like this :)

Good Luck

V.

**Reply by Zaedon33** [reply]
Thanks for your reply! All the module.conf files are in the etc/modules folder. If I move any .dist file from the folder I get the following error. Of course the error changes to the pertinent module name.

> Config::LoadFile: Failed open file '/home/zaedon/azeroth-server/etc/modules/AutoBalance.conf.dist'
> ConfigMgr::LoadModulesConfigs: Not found original config 'AutoBalance.conf.dist'. Stop loading

**Comment by Yehonal** [comment]
.dist files are currently used as fallback values and should always exist
In fact you could create .conf file with only the configurations you want to change. Thus having the others to be updated automatically if we change them overtime

**Reply by Zaedon33** [reply]
Thanks for the clarification!
